K2Bridge/Factories/BucketFactory.cs

```diff
@@ -23,16 +23,32 @@ public static DateHistogramBucket CreateFromDataRow(DataRow row)
         {
             Ensure.IsNotNull(row, nameof(row));
 
+            // fix column names
             var timestamp = row[(int)DateHistogramBucketColumnNames.Timestamp];
-            var count = row[(int)DateHistogramBucketColumnNames.Count];
+            var count = row["count_"];
```

Comment:
Do we need to add "count_" in a constant string definition or in DateHistogramsBucketColumnNames ?

Reply:
I would do it for consistency.

Reply:
Added count_ to DateHistogramsBucketColumnNames. As DateHistogramsBucketColumnNames contains now int and string, updated the enumeration to a class with readonly static property.